shared/provision/paper-key/index.tsx

```diff
@@ -1,24 +1,45 @@
 import * as React from 'react'
-import * as Constants from '../../constants/provision'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
+import {SignupScreen, errorBanner} from '../../signup/common'
 
 type Props = {
   onBack?: () => void
   onSubmit: (paperKey: string) => void
   hint: string
   error: string
+  waiting: boolean
 }
 
 class PaperKey extends React.Component<Props, {paperKey: string}> {
+  static navigationOptions = {
+    header: null,
+    headerBottomStyle: {height: undefined},
+    headerLeft: null, // no back button
+  }
+
   state = {paperKey: ''}
   _onSubmit = () => this.props.onSubmit(this.state.paperKey)
 
   render() {
     const props = this.props
 
     return (
-      <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} gap="medium">
+      <SignupScreen
+        banners={errorBanner(props.error)}
+        buttons={[
+          {
+            disabled: !this.state.paperKey,
+            label: 'Continue',
+            onClick: this._onSubmit,
+            type: 'Success',
+            waiting: props.waiting,
+          },
+        ]}
+        noBackground={true}
+        onBack={this.props.onBack}
+        title="Enter your paper key"
```

Comment:
Let's try to fit this without ellipsizing
<img width="383" alt="image" src="https://user-images.githubusercontent.com/11968340/64381094-67b4f280-d000-11e9-92d4-2d913ac7d5a9.png">